src/frontends/tensorflow/src/op/var_handle.cpp

```diff
@@ -0,0 +1,124 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
I expected you will take a look on this file first, because I have never wrote translators before. May I think you are ok with existing implementation?